.github/workflows/prod-api-CD.yml

```
@@ -42,26 +36,39 @@ jobs:
+            --repository-name "${{ env.ECR_APP_NAME }}" \
+            >/dev/null
 
       - name: Set up QEMU
         uses: docker/setup-qemu-action@v3
 
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
 
-      - name: Build & push API image
+      - name: Build and push API image
         uses: docker/build-push-action@v4
```

actionlint (1.7.12)

[error] the runner of "docker/build-push-action@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)